.gitignore

```diff
@@ -16,5 +16,5 @@ NashornProfile.txt
 **/JTreport/**
 **/JTwork/**
 /src/utils/LogCompilation/target/
-/.project/
+**.project**
```

Comment:
Changes to .gitignore should probably be made separately.

Reply:
Yes. This change should be removed from patch.